com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -229,26 +262,83 @@ protected override void Dispose(bool disposing)
         /// <inheritdoc />
         public override void Setup(ScriptableRenderContext context, ref RenderingData renderingData)
         {
+            SetupDebugRendering(context);
+
 #if ADAPTIVE_PERFORMANCE_2_0_0_OR_NEWER
             bool needTransparencyPass = !UniversalRenderPipeline.asset.useAdaptivePerformance || !AdaptivePerformance.AdaptivePerformanceRenderSettings.SkipTransparentObjects;
 #endif
             Camera camera = renderingData.cameraData.camera;
             ref CameraData cameraData = ref renderingData.cameraData;
             RenderTextureDescriptor cameraTargetDescriptor = renderingData.cameraData.cameraTargetDescriptor;
 
+            for (int i = 0; i < rendererFeatures.Count; ++i)
+            {
+                rendererFeatures[i].AddRenderPasses(this, ref renderingData);
+            }
+
+            int count = activeRenderPassQueue.Count;
+            for (int i = count - 1; i >= 0; i--)
+            {
+                if(activeRenderPassQueue[i] == null)
+                    activeRenderPassQueue.RemoveAt(i);
+            }
+
             // Special path for depth only offscreen cameras. Only write opaques + transparents.
+            SceneOverrides sceneOverride = DebugDisplaySettings.Instance.renderingSettings.sceneOverrides;
             bool isOffscreenDepthTexture = cameraData.targetTexture != null && cameraData.targetTexture.format == RenderTextureFormat.Depth;
-            if (isOffscreenDepthTexture)
+            bool sceneOverrideEnabled = sceneOverride != SceneOverrides.None;
+            if (isOffscreenDepthTexture || sceneOverrideEnabled)
```

Comment:
I'm not sure why we need to take this path. This path was just meant for rendering an offscreen depth only texture, we should probably ignore or restrict debugging in this situation.

Reply:
Interestingly, the Hackweek code was written like this - if I put the code back to the version on "master" then everything works except overdraw. I'll see if I can fix this a different way.